api/v1beta1/verticadb_types.go

```diff
@@ -964,6 +965,10 @@ const (
 	DefaultS3Region       = "us-east-1"
 	DefaultGCloudRegion   = "US-EAST1"
 	DefaultGCloudEndpoint = "https://storage.googleapis.com"
+
+	// VerticaDB.Spec.Annotation to enable the agent
+	StartAgent   = "start-agent"
+	AgentEnabled = "yes"
```

Comment:
Suggest a rename so that we know this corresponds to the annotation:
```suggestion
	RunAgentAnnotationEnabledValue = "yes"
```